### PR TITLE
fix(ui/components/left_panel): use links for accessibility

### LINKED ISSUE #40: Support shortcut keys on other platforms
I noticed that the shortcut button is ⌘ in the app and found it [in the code](https://github.com/kadirahq/storybook-ui/blob/2305d1f1109c38a104152fd47a3b681f969dd70c/src/modules/ui/components/left_panel/header.js#L52). It looks like it's hard coded. This makes me wonder whether shortcut keys are supported on other platforms. If so, I think it'd be appropriate to show a different icon in the button that's appropriate for the platform. Thoughts?

### PR DESCRIPTION
**Pull request by jmfrancois**
fix https://github.com/kadirahq/react-storybook/issues/505

### CODE REVIEW

**Comment by arunoda** [comment]
This looks pretty cool for me.
@mnmtanish Try to do a release, once you test it.

File: src/modules/ui/components/left_panel/stories.js [hunk 1]
```diff
@@ -49,49 +52,49 @@ class Stories extends React.Component {
...
     if (kind === selectedKind) {
       style.fontWeight = 'bold';
       return (
-        <div key={kind}>
-          <div
-            style={style}
-            onClick={this.fireOnKind.bind(this, kind)}
-          >
+        <li key={kind}>
+          <a style={style} onClick={onClick}>
```
Comment (frassinier) [inline src/modules/ui/components/left_panel/stories.js]: Should `<a href title="{`Open ${kind}`}" style={` be better?

Comment (mnmtanish) [inline src/modules/ui/components/left_panel/stories.js]: Yeah, I'll add them now

Comment (mnmtanish) [inline src/modules/ui/components/left_panel/stories.js]: Opened a PR but we need to verify whether improve accessibility before merging
https://github.com/kadirahq/storybook-ui/pull/44

**Pull request merged**